pkg/engine/request_performance_tests/play_game.go

```diff
@@ -0,0 +1,75 @@
+package requestperformancetests
+
+import (
+	"testing"
+
+	"github.com/YetAnotherSpieskowcy/Carcassonne-Engine/pkg/engine"
+	"github.com/YetAnotherSpieskowcy/Carcassonne-Engine/pkg/tilesets"
+)
+
+type TestedRequest func(games *[]engine.SerializedGameWithID, eng *engine.GameEngine, b *testing.B)
+
+func PlayGame(gameCount int, b *testing.B, testedRequest TestedRequest) {
```

Comment:
In all other functions `b *testing.B` is the last argument. For consistency, I'd advice to do it like so here as well

Reply:
consistenced